tests/unit/core/reducers/test_addons.js

```diff
@@ -82,4 +82,38 @@ describe('addon reducer', () => {
       },
     });
   });
+
+  it('does not use description from theme_data', () => {
+    // See: https://github.com/mozilla/addons-frontend/issues/2569
+    // TODO: Remove this when
```

Comment:
I don't think you need a TODO here. It's enough to put it in the offending code.